io.catenax.bom_as_specified/3.0.0/BomAsSpecified.ttl

```diff
@@ -0,0 +1,197 @@
+    samm:description "If the item is provided several times alike, the quantity on this level can specify how often. At this level pieces is assumed as the only unit. For further differentiation."@en;
+    samm:characteristic :QuantityCharacteristic;
+    samm:exampleValue "1"^^xsd:integer .
+
+:part a samm:Property;
+    samm:preferredName "part"@en;
+    samm:description "The description of the part in the primary language of the production facility of the product owner."@en;
+    samm:characteristic :PartCharacteristic .
+
+:IndexCharacteristic a samm:Characteristic;
+    samm:preferredName "index characteristic"@en;
+    samm:description "Characteristic of the index."@en;
+    samm:dataType xsd:integer .
+
+:QuantityCharacteristic a samm-c:Quantifiable;
+    samm:preferredName "quantity characteristic"@en;
+    samm:description "Characteristic of the quantity."@en;
+    samm:dataType xsd:integer;
+    samm-c:unit unit:piece .
+
+:PartCharacteristic a samm-c:Set;
+    samm:preferredName "part characteristic"@en;
+    samm:description "Characteristic of the part."@en;
+    samm:dataType :PartEntity .
+
+:PartEntity a samm:Entity;
+    samm:preferredName "part entity"@en;
+    samm:description "Entity encapsulating the properies describing a part"@en;
+    samm:properties (:ownerPartId [
+  samm:property :partVersion;
+  samm:optional "true"^^xsd:boolean
+] [
+  samm:property :partQuantity;
+  samm:optional "true"^^xsd:boolean
+] [
+  samm:property :partDescription;
+  samm:optional "true"^^xsd:boolean
+] [
+  samm:property :partClassification;
+  samm:optional "true"^^xsd:boolean
+]) .
+
+:ownerPartId a samm:Property;
+    samm:preferredName "owner part ID"@en;
+    samm:description "This is the key field of the component which usually keeps the part numbers used in after-sales, e.g. when repairing broken parts and searching for a replacement. This ownerPartId itself isn't usually bound to one part version, with the assumption that all part versions with the same ownerPartId are mutually interchangeable."@en;
+    samm:characteristic samm-c:Text;
+    samm:exampleValue "5A1BA91" .
+
+:partVersion a samm:Property;
+    samm:preferredName "part version"@en;
+    samm:description "This is the version of the part. The engineering will at times supercede an older part version by a newer one, which might have different material aspects, physical dimensions etc., still maintaining compatibility."@en;
+    samm:characteristic samm-c:Text;
+    samm:exampleValue "05" .
+
+:partQuantity a samm:Property;
+    samm:preferredName "part quantity"@en;
+    samm:description "This is the quantity how often this part is in the item."@en;
+    samm:characteristic :PartQuantityCharacteristic .
+
+:partDescription a samm:Property;
+    samm:preferredName "part description"@en;
+    samm:description "The description of the part in the primary language of the production facility of the product owner."@en;
+    samm:characteristic samm-c:Text .
```

Comment:
No description provided for simple type.

Reply:
@jSchuetz88  I need help here. Don't know what to do.

Reply:
Sorry, i meant "no value example provided"